test/getStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware, compose } from 'redux'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'redux'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->